runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/CloudDebuggerOptions.java

```diff
@@ -17,41 +17,41 @@
  */
 package org.apache.beam.runners.dataflow.options;
 
-import com.google.api.services.clouddebugger.v2.model.Debuggee;
-import org.apache.beam.sdk.annotations.Experimental;
 import org.apache.beam.sdk.options.Default;
 import org.apache.beam.sdk.options.Description;
 import org.apache.beam.sdk.options.Hidden;
 import org.apache.beam.sdk.options.PipelineOptions;
-import org.checkerframework.checker.nullness.qual.Nullable;
 
-/** Options for controlling Cloud Debugger. */
-@Description("[Experimental] Used to configure the Cloud Debugger")
-@Experimental
+/**
+ * Options for controlling Cloud Debugger.
+ *
+ * @deprecated Cloud debugger is no longer supported in this version of Beam. The cloud debugger
+ *     service is turning down (<a
+ *     href="https://cloud.google.com/debugger/docs/deprecations">announcement</a>). This interface
+ *     remains for purely back compatitive version and is subjected to remove in a future Beam
+ *     release.
+ */
+@Description("[Deprecated] Used to configure the Cloud Debugger")
+@Deprecated
 @Hidden
 public interface CloudDebuggerOptions extends PipelineOptions {
```

Comment:
May main concern is that will removing this option affecting users to upgrade their streaming pipeline to new version (even if they did not refer to related pipeline option)? Because for now DataflowPipelineOptions extends CloudDebuggerOptions.